tox_uv.toml

```diff
@@ -0,0 +1 @@
+no-binary = false
```

Comment:
@taurus-forever do we need to build from source for this repo? Artefacts released should only be the library itself.